CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+# v1.7.0 (2022-03-30)
+
+## OS Changes
+
+* Generate host rpm inventory to replace host container application inventory in SSM ([#1996])
```

Comment:
This might deserve an "above the line" callout like we did in v1.6.2:
"With this release, an inventory of software installed in Bottlerocket will now be reported to SSM, if the control container is in use."